--- saper-0.28-src/saper/stilview/stil.h.orig	Sun Dec  2 01:36:45 2001
+++ saper-0.28-src/saper/stilview/stil.h	Wed Sep 11 23:55:50 2002
@@ -16,9 +16,11 @@
 #ifndef _STIL_H
 #define _STIL_H
 
-#include <fstream.h>
+#include <fstream>
 #include "stilcomm.h"
 
+using namespace std;
+
 class STIL {
 
     public:
--- saper-0.28-src/saper/stilview/stil.cpp.orig	Sun Dec  2 01:36:44 2001
+++ saper-0.28-src/saper/stilview/stil.cpp	Wed Sep 11 23:56:34 2002
@@ -9,13 +9,15 @@
 #ifndef _STIL
 #define _STIL
 
-#include <iostream.h>
-#include <string.h>
-#include <fstream.h>
+#include <iostream>
+#include <string>
+#include <fstream>
 #include <stdio.h>      // For sprintf(), scanf() and NULL
 #include "stil.h"
 
-#define STILopenFlags (ios::in | ios::binary | ios::nocreate)
+using namespace std;
+
+#define STILopenFlags (ios::in | ios::binary)
 
 const float VERSION_NO = 2.16;
 
@@ -718,7 +718,7 @@
                 strncpy(dirs->dirName, line, j);
                 *((dirs->dirName)+j) = '\0';
 
-                dirs->position = inFile.tellg()-strlen(line)-1;
+                dirs->position = inFile.tellg()-(streamoff)(strlen(line)+1);
 
                 CERR_STIL_DEBUG << "getDirs() i=" << i << ", dirName=" << dirs->dirName << ", pos=" << dirs->position <<  endl;
 
@@ -853,7 +853,7 @@
 
     if (foundIt) {
         // Reposition the file pointer back to the start of the entry.
-        inFile.seekg(inFile.tellg()-strlen(line)-1);
+        inFile.seekg(inFile.tellg()-(streamoff)(strlen(line)+1));
         CERR_STIL_DEBUG << "pos2Entry() entry found" << endl;
         return true;
     }
